src/themes/components/modal.ts

```diff
@@ -0,0 +1,102 @@
+import { mode } from '@chakra-ui/theme-tools';
+import {
+  createMultiStyleConfigHelpers,
+  defineStyle,
+} from '@chakra-ui/styled-system';
+import { modalAnatomy as parts } from '@chakra-ui/anatomy';
+
+import { opacity } from '../shared/opacity';
+
+const { defineMultiStyleConfig, definePartsStyle } =
+  createMultiStyleConfigHelpers(parts.keys);
+
+const baseStyleOverlay = defineStyle({
+  bg: 'black',
+  zIndex: 'modal',
+  opacity: `${opacity[40]} !important`,
+});
+
+const baseStyleDialogContainer = defineStyle({
+  display: 'flex',
+  zIndex: 'modal',
+  justifyContent: 'center',
+  alignItems: 'center',
+  overflow: 'hidden',
+  overscrollBehaviorY: 'none',
+});
+
+const baseStyleDialog = defineStyle({
+  borderRadius: 'sm',
+  bg: mode('white', 'gray.700'),
+  color: 'inherit',
+  my: '16',
+  zIndex: 'modal',
+  boxShadow: mode('lg', 'dark-lg'),
+});
+
+const baseStyleHeader = defineStyle({
+  px: 'lg',
+  py: 'xl',
+  fontSize: 'md',
+  display: 'flex',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  fontWeight: 'semibold',
```

Comment:
I don't think our font supports this value. We only have bold and regular. Could you double check with our typography tokens? You can maybe use one of those instead defining each value

Reply:
as far as I can see in the design, we should use `heading3` styles